_capabilities/utdallas-lcn.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Learning Interpretable Cutset Networks"
+excerpt: "This library helps the user learn tractable, interpretable cutset networks (a type of probabilistic model which combines decision trees and tree Bayesian networks) from data. The learned networks can be used to answer various decision and explanation queries such as most probable explanation and estimating posterior marginal probability."
+tags: 
+- Analytics
+- Human-Machine Teaming
+- Explanation Framework
+
+submission_details:
+resources: 
+papers:
+- title: Look Ma, No Latent Variables: Accurate Cutset Networks via Compilation 
+url: https://proceedings.mlr.press/v97/rahman19a.html
+- title: Cutset Bayesian Networks: A New Representation for Learning Rao-Blackwellised Graphical Models
+url: https://doi.org/10.24963/ijcai.2019/797
```

Comment:
Since yaml uses colons for mappings, you'll need to use quotations for the two titles to prevent a syntax error:

```suggestion
- title: "Look Ma, No Latent Variables: Accurate Cutset Networks via Compilation"
url: https://proceedings.mlr.press/v97/rahman19a.html
- title: "Cutset Bayesian Networks: A New Representation for Learning Rao-Blackwellised Graphical Models"
url: https://doi.org/10.24963/ijcai.2019/797
```